examples/etcd/etcd-in-cluster/ocp/blueprint-v2/pv-etcd-backup.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: pv-volume-etcd
+  labels:
+    type: local
+spec:
+  storageClassName: default
+  capacity:
+    storage: 10Gi
```

Comment:
should we also specify that this needs to be changed based on the size of your etcd snapshot.